x/merkledb/db.go

```diff
@@ -1166,12 +1167,11 @@ func (db *merkleDB) invalidateChildrenExcept(exception *view) {
 // Otherwise leave [db.root] as Nothing.
 func (db *merkleDB) initializeRoot() error {
```

Comment:
The changes in this function aren't required... But I think they improve the code quality.